suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -117,27 +118,29 @@ export const TransactionListItem = memo(
                 onPress={() => handleNavigateToTransactionDetail()}
                 style={applyStyle(transactionListItemStyle, { isFirst, isLast })}
             >
-                <TransactionListItemIcon
-                    cryptoIconName={transaction.symbol}
-                    transactionType={transaction.type}
-                />
-                <Box>
-                    <Box style={applyStyle(titleCointainerStyle)}>
-                        <Text>{transactionTypeProperties.prefix}</Text>
-                        <Box style={applyStyle(addressStyle)}>
-                            <Text
-                                variant="label"
-                                color="gray600"
-                                numberOfLines={1}
-                                ellipsizeMode="middle"
-                            >
-                                {transaction.txid}
-                            </Text>
+                <Box style={applyStyle(descriptionBoxStyle)}>
+                    <TransactionListItemIcon
+                        cryptoIconName={transaction.symbol}
+                        transactionType={transaction.type}
+                    />
+                    <Box marginLeft="medium" flex={1}>
+                        <Box flexDirection="row" justifyContent="center">
+                            <Text>{transactionTypeProperties.prefix}</Text>
+                            <Box style={applyStyle(addressStyle)}>
+                                <Text
+                                    variant="label"
+                                    color="gray600"
+                                    numberOfLines={1}
+                                    ellipsizeMode="middle"
+                                >
+                                    {transaction.txid}
```

Comment:
I think we have this wrong all the time. `txid` is not equal from/to I think.

Reply:
True. I didn't notice that. So there should be an input address for incoming transitions and an output address for outcoming transactions. What about the other types as `self`, `joint`, `failed` etc.?

Reply:
I think there should be output/input for every type of transaction

Reply:
It was changed in 4999afd8d to show the first transaction targets as `transaction.targets[0].addresses?.[0]`. Could you let me know if this is the correct solution?

In suite-desktop, they show all target addresses, not only the first one: https://github.com/trezor/trezor-suite/blob/d8b77aafb150766efa0c348f6e2b85299371e6dc/packages/suite/src/components/wallet/TransactionItem/components/TargetAddressLabel.tsx#L32-L43

Reply:
We will have to create something like AddressFormatter, which will show for example only first adress (or any adress with label and onPress it will show some modal with other outputs). There is task for this but some parts need to be refined with @mnuky

Reply:
I'm not sure if I understand correctly the issue here. There is a improvement planned for shorten transaction IDs and addressed to be "clickable" and if user taps on it, toaster with full address appears. 
![Screenshot 2023-02-06 at 13 07 02](https://user-images.githubusercontent.com/84183817/216967263-fb1f198e-69fb-49a5-938f-cc6acc7b880c.png)